packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.js

```diff
@@ -255,6 +272,16 @@ export default class QuanticSort extends LightningElement {
     return this.hasInitializationError || !!this.errorMessage;
   }
 
+  get sortContainerCSSClass() {
+    return this.variant === this.variants.default.name
+      ? 'sort__container'
+      : undefined;
+  }
```

Comment:
Put this at the top of this file, even outside the LightningElement

```
const SORT_VARIANTS = {
  DEFAULT: 'default',
  WIDE: 'wide',
};
Object.freeze(SORT_VARIANTS); // This means it can never be modified after this.
```
And then you can use

```suggestion
  get sortContainerCSSClass() {
    return this.variant === SORT_VARIANTS.DEFAULT
      ? 'sort__container'
      : '';
  }
```